utils/threshAlgos.py

```diff
@@ -472,39 +474,69 @@ def calibrateThrDAC(inputFile,fitRange="0,255",listOfVFATs=None,noLeg=False,outp
         savePlots     - Make a '*.png' file for all plots that will be saved in the output TFile
     """
```

Comment:
You need to update the function documentation to indicate what `args` is and what fields are expected.

Reply:
done